docs/en/install_deploy.md

```diff
@@ -0,0 +1,393 @@
+* Modify `zk_cluster` to the address of the zk cluster that has been started. Ip is the ip of the machine where zk is located, and port is the port number configured by clientPort in the zk configuration file. If zk is in cluster mode, separate it with commas, and the format is ip1:port1,ip2:port2, ip3:port3.
+* If you share zk with other OpenMLDB, you need to modify `zk_root_path`.
+
+```
+--endpoint=172.27.128.31:6527
+--zk_cluster=172.27.128.33:7181,172.27.128.32:7181,172.27.128.31:7181
+--zk_root_path=/openmldb_cluster
+--enable_distsql=true
+```
+
+**Notice:** The endpoint cannot use 0.0.0.0 and 127.0.0.1.
+
+#### 3 Start the service
+
+```
+sh bin/start.sh start nameserver
+```
+
+Repeat the above steps to deploy multiple nameservers.
+
+#### 4 Verify the running status of the service
+
+```bash
+$ ./bin/openmldb --zk_cluster=172.27.128.31:7181,172.27.128.32:7181,172.27.128.33:7181 --zk_root_path=/openmldb_cluster --role=ns_client
+> shown
+  endpoint role
+-----------------------------
+  172.27.128.31:6527 leader
+```
+
+### Deploy APIServer
+
+APIServer is responsible for receiving http requests, forwarding them to OpenMLDB and returning results. It is stateless and is not a must-deploy component of OpenMLDB.
+Before running, make sure that the OpenMLDB cluster has been started, otherwise APIServer will fail to initialize and exit the process.
+
+#### 1 Download the OpenMLDB deployment package
+
+```
+wget https://github.com/4paradigm/OpenMLDB/releases/download/v0.4.3/openmldb-0.4.3-linux.tar.gz
+tar -zxvf openmldb-0.4.3-linux.tar.gz
+mv openmldb-0.4.3-linux openmldb-apiserver-0.4.3
+cd openmldb-apiserver-0.4.3
+```
+
+#### 2 Modify the Configuration File: conf/apiserver.flags
+
+* Modify `endpoint`. The endpoint is the deployment machine ip/domain name and port number separated by colons.
+* Modify ``zk_cluster`` to the zk cluster address of OpenMLDB to be forwarded to.
+
+```
+--endpoint=172.27.128.33:8080
+--role=apiserver
+--zk_cluster=172.27.128.33:7181,172.27.128.32:7181,172.27.128.31:7181
+--zk_root_path=/openmldb_cluster
+--openmldb_log_dir=./logs
+```
+
+**Notice:**
+* The endpoint cannot use 0.0.0.0 and 127.0.0.1. You can also choose not to set `--endpoint`, and only configure the port number `--port`.
+* You can also configure the number of threads of APIServer, `--thread_pool_size`, the default is 16.
+
+#### 3 Start the service
+
+```
+sh bin/start.sh start apiserver
+```
+
+**Notice:** If the core is dropped when the nameserver/tablet/apiserver is started through the release package on the Linux platform, it is very likely that the instruction set is incompatible, and you need to compile openmldb through the source code. For source code compilation refer to [here](./compile.md), you need to use method 3 to compile the complete source code.
+
+### Deploy TaskManager
+
+#### 1 Download the OpenMLDB deployment package and the Spark distribution optimized for feature engineering
```

Comment:
#### 1 Download the OpenMLDB Spark distribution that is optimized for feature engineering

Reply:
Ok, thank you.